src/test/app/AMM_test.cpp

```diff
@@ -1013,8 +1051,8 @@ struct AMM_test : public jtx::AMMTest
             ammAlice.deposit(bob, XRP(10));
             ammAlice.deposit(
                 bob,
-                XRP(1'000),
-                std::nullopt,
+                gw["USD"](100),
```

Comment:
Shouldn't we keep the original test with XRP and null USD?

Reply:
accidentally changed it. just updated it back